include/libcyphal/transport/session.hpp

```diff
@@ -14,7 +14,7 @@ namespace libcyphal
 namespace transport
 {
 
-class ISession : public IRunnable
```

Comment:
Class docs for this is important. Describe what a session is in libcyphal.

Reply:
I believe it will be gone completely. But you are right, I will describe ALL other session interfaces.